Libraries/LibWeb/CSS/StyleValues/CSSColorFunction.h

```diff
@@ -11,11 +11,11 @@
 namespace Web::CSS {
 
 // https://drafts.css-houdini.org/css-typed-om-1/#csscolor
-class CSSColor final : public CSSColorValue {
+class CSSColorFunction final : public CSSColorValue {
```

Comment:
As you're renaming this, could you make it something like `ColorFunctionStyleValue`? The reason it was `CSSColor` is because that's the name of a CSSOM type - but we've since realised that our style values don't want to be CSSOM types.

Renaming all the "named after a CSSOM type" style value classes has been on my list for a while but I keep putting it off because it sounds like a conflict magnet. :sweat_smile: